components/OurRecruiter.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from 'react';
+import Image from 'next/image';
+
+import image3 from '@assets/banner/ankur.jpg';
+import image2 from '@assets/banner/ashpreet.jpg';
+import image1 from '@assets/banner/ourrecuiters.jpg';
+
+const OurRecruiter: React.FC = () => {
+  const slides = [
+    { id: 1, image: image1, alt: 'Recruiter Portrait' },
+    { id: 2, image: image3, alt: 'Placement Drive' },
+    { id: 3, image: image2, alt: 'Campus Interaction' },
+  ];
```

Comment:
change import order and name accordingly before this fix

```suggestion
    { id: 1, image: image1, alt: 'Recruiter Portrait' },
    { id: 2, image: image2, alt: 'Placement Drive' },
    { id: 3, image: image3, alt: 'Campus Interaction' },
  ];
```